python/requirements.txt

```diff
@@ -1,17 +1,17 @@
-gekko==0.2.8
-networkx==2.6.2
-python-Levenshtein==0.12.1
+defusedxml==0.7.1
 elasticsearch==8.4.3
+gekko==0.2.8
 gensim==4.3.3
-torch==1.12.0
-six==1.16.0
-torchmetrics==0.9.3
-igraph==0.10.2
-scikit-learn==0.24.2
 gqlalchemy==1.4.1
+igraph==0.10.2
 mysql-connector-python==8.0.32
-oracledb==1.2.2
-pyodbc==4.0.35
+networkx==2.6.2
```

Comment:
E.g. NetworkX latest version is 3.4.2 -> go over this list again, and try to bump everything

The only concern I have are these database connector like `mysql-connector-python` because that might be sensitive to the version of databases our customers are using... 🤔 But, I would still bump that, if there is a problem for a given customer we can manually downgrade directly inside the Docker image.